src/main/java/org/dependencytrack/tasks/PolicyEvaluationTask.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * This file is part of Dependency-Track.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) Steve Springett. All Rights Reserved.
+ */
+package org.dependencytrack.tasks;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.dependencytrack.event.PolicyEvaluationEvent;
+import org.dependencytrack.event.ProjectMetricsUpdateEvent;
+import org.dependencytrack.model.Component;
+import org.dependencytrack.model.Project;
+import org.dependencytrack.policy.PolicyEngine;
+import alpine.common.logging.Logger;
+import alpine.event.framework.Event;
+import alpine.event.framework.Subscriber;
+
+public class PolicyEvaluationTask implements Subscriber {
+
+    private static final Logger LOGGER = Logger.getLogger(PolicyEvaluationTask.class);
+
+    /**
+     * {@inheritDoc}
+     */
+    public void inform(final Event e) {
+        if (e instanceof PolicyEvaluationEvent) {
+            final PolicyEvaluationEvent event = (PolicyEvaluationEvent) e;
+            LOGGER.info("Starting policy evaluation");
```

Comment:
This log message by itself is not really helpful. When a project is being evaluated, the project UUID should be included. If the entire portfolio is evaluated, the log messages should state that. For individual components, log messages like this will probably produce too much noise, so it'd be better to log in `DEBUG` for those, if at all.